src/components/AccountIcon.js

```diff
@@ -17,54 +17,35 @@
 'use strict';
 
 import PropTypes from 'prop-types';
```

Comment:
I've used hooks now in the substrate re-factoring for this file here: https://github.com/paritytech/parity-signer/pull/293/files#diff-4f18afcbf235db6a111ab00e2a6bee19L24

Reply:
Nice!